website/source/docs/platform/k8s/run.html.md

```diff
@@ -67,7 +67,25 @@ $ helm install --name vault ./
 ...
 ```
 
-_That's it._ The Helm chart does everything to setup a Vault-on-Kubernetes deployment.
+!> **IMPORTANT NOTE:** Vault Helm will not initialize and unseal Vault out of the box.  
```

Comment:
Nit: I worry that "out of the box" may be an expression that could be misinterpreted here. I would prefer `automatically`, but I'll defer to you

Reply:
Done!